packages/ui-badge/src/Badge/README.md

```diff
@@ -78,40 +79,57 @@ Setting `type="notification"` will render small circles that should not contain
 example: true
 ---
 <div>
-  <Badge standalone count={6} margin="0 small 0 0" />
-  <Badge standalone variant="success" count={12} margin="0 small 0 0"  />
-  <Badge standalone variant="danger" count={18} countUntil={10} margin="0 small 0 0" />
-  <Badge
-    type="notification"
-    standalone
-    formatOutput={function () {
-      return <ScreenReaderContent>This is a notification</ScreenReaderContent>
-    }}
-    margin="0 small 0 0"
-  />
-  <Badge
-    variant="success"
-    type="notification"
-    standalone
-    formatOutput={function () {
-      return <ScreenReaderContent>This is a success notification</ScreenReaderContent>
-    }}
-    margin="0 small 0 0"
-  />
-  <Badge
-    variant="danger"
-    type="notification"
-    standalone
-    formatOutput={function () {
-      return <ScreenReaderContent>This is a danger notification</ScreenReaderContent>
-    }}
-  />
+  <Flex padding='small' display='inline-flex' alignItems="center">
+    <Badge standalone count={6} margin='0 small 0 0' />
+    <Badge
+      type="notification"
+      standalone
+      formatOutput={function () {
```

Comment:
Could you refactor these to arrow functions? For sake of consistency

Reply:
Sure!